service/src/main/java/bio/terra/pipelines/db/entities/PipelineInputDefinition.java

```diff
@@ -27,6 +27,12 @@ public class PipelineInputDefinition extends BasePipelineVariableDefinition {
   @Column(name = "default_value")
   private String defaultValue; // must be a String representation of the value
 
+  @Column(name = "min_value")
+  private Integer minValue; // inclusive minimum
```

Comment:
are we sure that min/max values will always be integers?

Reply:
yea i wasnt sure about this either, i looked up some [postgres types](https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-FLOAT) and it seems `DOUBLE PRECISION` would be the best type to use?  Happy to change it but I didnt have a non integer use case on top of mind so i just went with integer since its simpler.  I think this double type woudl be fine too